static/app/components/keyValueData/index.tsx

```diff
@@ -289,6 +299,8 @@ const ValueSection = styled('div')<{hasEmptySubject: boolean; hasErrors: boolean
 const ValueWrapper = styled('div')<{hasSuffix: boolean}>`
   word-break: break-word;
   grid-column: ${p => (p.hasSuffix ? 'span 1' : '1 / -1')};
+  max-width: 100%;
+  overflow: hidden;
```

Comment:
What is this for? We rarely ever want `overflow: hidden` when it comes to rendering data.